### PR TITLE
b3sum: add page

### PR DESCRIPTION
**Pull request by isaak-pdm**
I like b3sum and use tldr pages. So I thought I would start contributing with writing the missing tldr page for b3sum. If anything is wrong please tell me/suggest changes.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** b3sum 1.3.3

### CODE REVIEW

File: pages/common/b3sum.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# b3sum
+
+> A command line utility for calculating BLAKE3 hashes, similar to Coreutils tools like b2sum or md5sum.
```
Comment (kbdharun) [inline pages/common/b3sum.md:3]:
```suggestion
> Command line utility for calculating BLAKE3 hashes.
> It is similar to Coreutils tools like b2sum or md5sum.
```
Would this be better?

File: pages/common/b3sum.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Calculate BLAKE3 checksums for multiple files:
+
+`b3sum {{path/to/file1}} {{path/to/file2}}`
+
+- Calculate the BLAKE3 checksum from `stdin`:
+
+`{{some_command}} | b3sum`
+
+- Create a file of BLAKE3 sums:
```
Comment (kbdharun) [inline pages/common/b3sum.md:18]:
```suggestion
- Create a file of BLAKE3 checksums:
```

File: pages/common/b3sum.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`b3sum {{path/to/file}} > {{path/to/file.b3}}`
+
+- Read and verify a file of BLAKE3 sums:
```
Comment (kbdharun) [inline pages/common/b3sum.md:22]:
```suggestion
- Read and verify a file of BLAKE3 checksums:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by isaak-pdm** [comment]
You are right. I appreciate your improvements and accepted them. Thanks @kbdharun

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**